packages/astro/src/overlay/index.ts

```diff
@@ -0,0 +1,30 @@
+import type { DevOverlayPlugin } from 'astro';
+
+import * as Spotlight from '@spotlightjs/core';
+
+const sentryLogo = `<svg class="css-lfbo6j e10nushx4" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 44" width="400" height="352"
```

Comment:
I simply inlined this because TS complained, and I figured it's not worth it to figure out how to fix it xD